be/src/storage/binlog_reader.cpp

```diff
@@ -0,0 +1,324 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "storage/binlog_reader.h"
+
+#include <utility>
+
+#include "column/datum.h"
+#include "storage/chunk_helper.h"
+#include "storage/rowset/rowid_range_option.h"
+#include "storage/rowset/segment_options.h"
+#include "storage/tablet.h"
+
+namespace starrocks {
+
+std::unordered_set<std::string_view> BINLOG_META_SET{BINLOG_OP, BINLOG_VERSION, BINLOG_SEQ_ID, BINLOG_TIMESTAMP};
```

Comment:
should the global variable be named as `gXxxYyy` ?

Reply:
1. is it safe to use a `string_view` ? 
2. why not maintain this state at BinlogManager ?

Reply:
This variable is not very necessary, and just remove it.